src/acceleration/AitkenAcceleration.hpp

```diff
@@ -36,6 +36,11 @@ class AitkenAcceleration : public Acceleration {
   virtual void iterationsConverged(
       const DataMap &cpldata);
 
+protected:
```

Comment:
Could go to `private` here, no?

Reply:
Yes. However looking at Benni's structure I think it is nicer to keep this as protected, since it now overrides concatenateCouplingData in acceleration.hpp.